src/SearchResults/index.js

```diff
@@ -420,6 +420,8 @@ SearchResults.prototype.getFacetByName = function(name) {
     find(this.hierarchicalFacets, predicate);
 };
 
+var EMPTY_NORMALIZED_FACET_VALUES = (function() { var a = []; a.facetType = 'conjunctive'; return a; })();
```

Comment:
> Why do we need to attach a property containing facetType to the array here? why not just return an empty array?

Ok I saw the other lines in the helper, I did not know we had this in the helper. Is it something we use in some implementations? (facetType)